src/SmartComponents/Drift/DriftDropDown.js

```diff
@@ -0,0 +1,52 @@
+
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { translateDriftDropdownItems } from '../../AppConstants';
+
+import {
+    Dropdown,
+    DropdownToggle,
+    DropdownItem
+} from '@patternfly/react-core';
+import CaretDownIcon from '@patternfly/react-icons/dist/esm/icons/caret-down-icon';
+
+export const DriftDropDown = ({ fetchDriftData, selectedFilter }) => {
+
+    const intl = useIntl();
+    const dropdownItems = useRef([]);
+    const [isOpen, setIsOpen] = useState(false);
+
+    useEffect(() => {
+        dropdownItems.current = translateDriftDropdownItems(intl);
+    }, []);
+
+    const onItemClick = (item) => {
+        fetchDriftData(item);
+        setIsOpen((prevOpen) => !prevOpen);
```

Comment:
Looks like we forgot to call the `focus`
```suggestion
        setIsOpen((prevOpen) => !prevOpen);
        item.el.focus();
```